### PR TITLE
Flow Updates

### PR DESCRIPTION
**Pull request by mhussaincgi**
1- Deactivate following flows

Parent flow for Case
Parent flow for Account
Parent flow for HCC

2- Create new flow for Account by optimizing existing flows
3- Update trigger condition for Update Case fields (record Triger Flow) Entry conditions.

### CODE REVIEW

File: TPL/force-app/main/default/flows/Parent_Flow_For_Account.flow-meta.xml [hunk 1]
```diff
@@ -219,7 +219,7 @@
         <recordTriggerType>CreateAndUpdate</recordTriggerType>
         <triggerType>RecordAfterSave</triggerType>
     </start>
-    <status>Active</status>
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/flows/Parent_Flow_For_Account.flow-meta.xml:222]: Authorized deactivation of replaced flows; will approve destruction changes after testing processes and final deployments are completed.

File: TPL/force-app/main/default/flows/Update_Account_Fields.flow-meta.xml [hunk 2]
```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/flows/Update_Account_Fields.flow-meta.xml:1]: This is a simplified version of the account flow which is stand-alone and no longer executes sub-flows.

**Review by cgijeffolsen — APPROVED** [review]
@mhussaincgi @NataliaNikishina Approved, thank you.

**Review by NataliaNikishina — APPROVED** [review]
@mhussaincgi approved

**Pull request merged**